### ISSUE TITLE
Github Action: Publish to DockerHub on push to master

### ISSUE THREAD

**Issue by XiXiaPdx** [AntonEmery/chord-app#41]
Reference research in #40 

Add step to current NoOp action to publish to dockerhub.

1. Need docker hub username and password entered into secrets. @AntonEmery , can you provide that please? 
2. Use official dockhub action for this - https://github.com/marketplace/actions/build-and-push-docker-images#usage

Dockerhub link - https://hub.docker.com/r/antonemery/chord-app

**Comment by XiXiaPdx** [comment]
Have a feeling I'll need to delete some test images during this process. Sounds like removing a tag will remove all images with that tag?

https://devopsheaven.com/docker/dockerhub/2018/04/09/delete-docker-image-tag-dockerhub.html

**Comment by XiXiaPdx** [comment]
I forked this repo and have validated an action that can build and push the image to docker hub.  @AntonEmery  , when you get the secrets added  (or when I can), I'll PR the changes 

```
  username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

**Comment by XiXiaPdx** [comment]
closed by #44
